portal/server/graphql/queries/deals-query.js

```diff
@@ -0,0 +1,42 @@
+const gql = require('graphql-tag');
+
+const dealsQuery = gql`
+query Deals($start: Int, $pagesize: Int, $filters:[DashboardReportsFilters]){
+  deals(params: {start: $start, pagesize: $pagesize, filters: $filters}) {
+    count
+    deals {
+      _id
+      submissionType
+      bankInternalRefName
+      additionalRefName
+      updatedAt
+      status
+      bank {
+        name
+      }
+      details {
+        ukefDealId
+        maker {
+          username
+          firstname
+          surname
+        }
+        checker {
+          username
+          firstname
+          surname
+        }
+        checkerMIN {
+          username
+          firstname
+          surname
+        }
+        submissionDate
+        approvalDate
+        created
+      }
+    }
+  }
+}`;
```

Comment:
is adding this query intentional?

Reply:
I thought you added this :D

Reply:
😂 OK either way we can remove it, it's now a simpler all-deals-query.js

Reply:
Lots of PRs merged since this was opened 🙃

Reply:
I can take it out. there is one failing test that I need to fix anyway

Reply:
the beautiful part of this is that I can't see it in any of the commits :|

Reply:
Amazing 😂